examples/bin/supervise

```diff
@@ -179,13 +195,28 @@ usage() unless GetOptions(
   'vardir|d=s',
   'kill-timeout|t=i',
   'chdir=s',
-  'svlogd:s'
+  'svlogd:s',
+  'array|a=s{,}'
```

Comment:
Nit: maybe rename this to something like `commands`

Reply:
`commands` is already a declared var in the script. I can rename it to `command_array`